### PR TITLE
Added gcloud-ga-latest/wrapper-release E2E tests

### PR DESCRIPTION
**Pull request by zoran15**
Added gcloud-ga-latest/wrapper-release E2E tests for all commands except beta-only ones at this time (OS upgrade and one-step-import). 

The direct reason for adding these tests is to ensure gcloud GA works as expected once regionalized wrapper images are rolled out to beta. In the long term, it's still a good idea to test gcloud GA (not beta like existing tests) against release wrappers as this is what our customers use predominantly.

### CODE REVIEW

File: cli_tools_e2e_test/common/utils/utils.go [hunk 1]
```diff
@@ -35,9 +35,10 @@ type CLITestType string
 
 // List all test types here
 const (
-	Wrapper                   CLITestType = "1 wrapper"
-	GcloudProdWrapperLatest   CLITestType = "2 gcloud-prod wrapper-latest"
-	GcloudLatestWrapperLatest CLITestType = "3 gcloud-latest wrapper-latest"
+	Wrapper                       CLITestType = "1 wrapper"
+	GcloudBetaProdWrapperLatest   CLITestType = "2 gcloud-beta-prod wrapper-latest"
```
Comment (dntczdx) [inline cli_tools_e2e_test/common/utils/utils.go:39]: The format like gcloud-beta-prod is a bit hard to understand. Maybe find a a more clear way to describe it,

Comment (zoran15) [reply cli_tools_e2e_test/common/utils/utils.go:39]: The format is <command>-<release track>-<code branch> which more precisely indicate what is being tested than only <command>-<code branch> as we had before. 

Do you have a suggestion how to make it easier to understand?

**Review by EricEdens — APPROVED** [review]

File: cli_tools_e2e_test/common/utils/utils.go (hunk 1 above)
Comment (dntczdx) [reply cli_tools_e2e_test/common/utils/utils.go:39]: gcloud(beta)-prod lokks more clear to me

Comment (zoran15) [reply cli_tools_e2e_test/common/utils/utils.go]: Changed

**Review by dntczdx — APPROVED** [review]

**Pull request merged**